### DISCUSSION TITLE
Request--Add iGOS++ method to pytorch-grad-cam

### DISCUSSION THREAD

**Discussion by mingqiJ** [jacobgil/pytorch-grad-cam#362] [Q&A]
Hi

I created this discussion to ask if someone could add our **XAI method iGOS++** to pytorch-grad-cam.

This is a state of the art method for Explainable AI for computer vision. **It has better insert and delete scores than many other existing method owners, like Score-CAM, Grad-CAM++, etc.**

This is the Github link: https://github.com/khorrams/IGOS_pp

It contains iGOS++ and iGOS(The method of the previous generation) method. 

Others are certainly welcome to see and use this method. iGOS++ is much more advanced than other methods. I'm sure you'll be interested.

If so, I would really appreciate it.

**Comment by jacobgil** [comment]
Yes, certainly.
A pull request will be most welcome.
I can work on adding this myself, but it might take me a few weeks to get to this.

**Reply by mingqiJ** [reply]
**Thank you so much!!** I believe you will like this method, its performance is very excellent. Look forward to iGOS++ appearing on this repo. Hopping more people know about our method.

**In the meantime, if you have any questions about the code, please feel free to contact me. Anytime.**
